tests/load/pipeline/test_filesystem_pipeline.py

```diff
@@ -224,10 +224,7 @@ def some_source():
 
 
 @pytest.mark.essential
-def test_delta_table_core(
-    default_buckets_env: str,
-    local_filesystem_pipeline: dlt.Pipeline,
-) -> None:
+def test_delta_table_core(default_buckets_env: str) -> None:
```

Comment:
generally speaking you should be able to use our destinationconfigs here..

Reply:
The `destination_configs` method needs some extra logic to work with `table_format`. I already wrote the code for it in another branch that adds Delta `merge` support. I'll merge that branch first, and then refactor this to use destination configurations.